src/components/404/NotFoundSideBar.js

```diff
@@ -0,0 +1,98 @@
+import React from "react";
+
+class NotFoundSideBar extends React.Component {
+  render() {
```

Comment:
this is a statelless component please change this. if you have eslint set up properly you would see the error